package.json

```diff
@@ -1,125 +1,127 @@
+  "description": "Use MyST in JupyterLab",
+  "keywords": [
+    "jupyter",
+    "jupyterlab",
+    "jupyterlab-extension"
+  ],
+  "homepage": "https://github.com/executablebooks/jupyterlab-myst",
+  "bugs": {
+    "url": "https://github.com/executablebooks/jupyterlab-myst/issues"
+  },
+  "license": "MIT",
+  "author": {
+    "name": "Executable Book Project",
+    "email": "executablebooks@gmail.com"
+  },
+  "files": [
+    "lib/**/*.{d.ts,eot,gif,html,jpg,js,js.map,json,png,svg,woff2,ttf}",
+    "style/**/*.{css,js,eot,gif,html,jpg,json,png,svg,woff2,ttf}",
+    "style/index.js"
+  ],
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "style": "style/index.css",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/executablebooks/jupyterlab-myst.git"
+  },
+  "scripts": {
+    "build": "jlpm build:css && jlpm build:lib && jlpm build:labextension:dev",
+    "build:css": "tailwindcss -m -i ./style/tailwind.css -o style/app.css",
+    "build:labextension": "jupyter labextension build .",
+    "build:labextension:dev": "jupyter labextension build --development True .",
+    "build:lib": "tsc --sourceMap",
+    "build:lib:prod": "tsc",
+    "build:prod": "jlpm clean && jlpm build:lib:prod && jlpm build:labextension",
+    "clean": "jlpm clean:lib",
+    "clean:all": "jlpm clean:lib && jlpm clean:labextension && jlpm clean:lintcache",
+    "clean:labextension": "rimraf jupyterlab_myst/labextension jupyterlab_myst/_version.py",
+    "clean:lib": "rimraf lib tsconfig.tsbuildinfo",
+    "clean:lintcache": "rimraf .eslintcache .stylelintcache",
+    "eslint": "jlpm eslint:check --fix",
+    "eslint:check": "eslint . --cache --ext .ts,.tsx",
+    "install:extension": "jlpm build",
+    "lint": "jlpm stylelint && jlpm prettier && jlpm eslint",
+    "lint:check": "jlpm stylelint:check && jlpm prettier:check && jlpm eslint:check",
+    "prettier": "jlpm prettier:base --write --list-different",
+    "prettier:base": "prettier \"**/*{.ts,.tsx,.js,.jsx,.css,.json,.md}\"",
+    "prettier:check": "jlpm prettier:base --check",
+    "stylelint": "jlpm stylelint:check --fix",
+    "stylelint:check": "stylelint --cache \"style/**/*.css\"",
+    "test": "jest --coverage",
+    "watch": "run-p watch:src watch:labextension",
+    "watch:css": "tailwindcss -w -i ./style/tailwind.css -o style/app.css",
+    "watch:labextension": "jupyter labextension watch .",
+    "watch:src": "tsc -w"
+  },
+  "dependencies": {
+    "@jupyterlab/application": "^4.0.0",
+    "@jupyterlab/codeeditor": "^4.0.0",
+    "@jupyterlab/markdownviewer": "^4.0.0",
+    "@jupyterlab/notebook": "^4.0.0",
+    "@jupyterlab/rendermime": "^4.0.0",
+    "@jupyterlab/translation": "^4.0.0",
+    "@myst-theme/diagrams": "^0.2.6",
+    "@myst-theme/frontmatter": "^0.2.6",
+    "@myst-theme/providers": "^0.2.6",
+    "katex": "^0.15.2",
+    "myst-ext-card": "^0.0.7",
+    "myst-ext-grid": "^0.0.7",
+    "myst-ext-proof": "^0.0.2",
+    "myst-ext-tabs": "^0.0.7",
```

Comment:
We can bring the new stuff forward pretty easy. If we are calling this the new "main" let me know, and I can rebase the changes over here.